unity/Assets/Scripts/BaseFPSAgentController.cs

```diff
@@ -448,6 +457,41 @@ public void SetAgentMode(string mode)
                 // limit camera from looking too far down
                 this.horizonAngles = new float[] { 30.0f, 0.0f, 330.0f };
             }
+
+            else if(whichMode == "drone")
+            {
+                //toggle first person character cull
+                fpcc.SwitchRenderersToHide(whichMode);
+
+                VisibilityCapsule = DroneVisCap;
+                m_CharacterController.center = new Vector3(0,0,0);
+                m_CharacterController.radius = 0.2f;
```

Comment:
should this magic constant be in the prefab of the drone?

Reply:
I dont think so, the drone isn't a separate prefab, it's a modification of the FPSController prefab that toggles on all the drone settings. Both the "tall" and"bot" modes set the center and radius to the appropriate values for their modes as well.

Reply:
got it

Reply:
I guess they could be in the DroneAgentController class if you inherited PhysicsFPSController

Reply:
yes agreed, I will move them there